components/registry-facade-api/go/config/config.go

```diff
@@ -80,10 +81,11 @@ type RedisCacheConfig struct {
 
 	SingleHostAddress string `json:"singleHostAddr,omitempty"`
 
-	MasterName    string   `json:"masterName,omitempty"`
-	SentinelAddrs []string `json:"sentinelAddrs,omitempty"`
-	Username      string   `json:"username,omitempty"`
-	Password      string   `json:"-" env:"REDIS_PASSWORD"`
+	Username string `json:"username,omitempty"`
+	Password string `json:"-" env:"REDIS_PASSWORD"`
```

Comment:
With the env `REDIS_` prefix means we implicitly support the Redis cluster only.

Reply:
Yes, or a cloud service.

Note: we are switching to GCP memorystore.